### PR TITLE
Release skeleton

### PR DESCRIPTION
**Pull request by lucafavatella**
In scope of https://www.pivotaltracker.com/story/show/163184795

### CODE REVIEW

**Comment by lucafavatella** [comment]
@tolbrino What is the plan with https://github.com/aeternity/aeternity/blob/f704c90f439db057816b919c447e236c1d8de6c5/appveyor.yml#L1 ?

**Comment by tolbrino** [comment]
> @tolbrino What is the plan with
> 
> [aeternity/appveyor.yml](https://github.com/aeternity/aeternity/blob/f704c90f439db057816b919c447e236c1d8de6c5/appveyor.yml#L1)
> 
> Line 1 in [f704c90](/aeternity/aeternity/commit/f704c90f439db057816b919c447e236c1d8de6c5)
> 
>  version: 1.3.0-{branch}-{build} 
> ?

I guess it might make more sense to just remove the version prefix and keep branch + build number.

**Comment by lucafavatella** [comment]
[CI test job](https://circleci.com/gh/aeternity/aeternity/1830) failed with unrelated error:
```
%%% aehttp_integration_SUITE ==> all.external_endpoints.oracle_transactions: FAILED
%%% aehttp_integration_SUITE ==> {max_reached,
    [{aecore_suite_utils,mine_blocks_until_txs_on_chain,4,
         [{file,
              "/home/builder/aeternity/_build/test/lib/aecore/test/aecore_suite_utils.erl"},
          {line,270}]},
     {aehttp_integration_SUITE,wait_for_tx_hash_on_chain,1,
         [{file,
              "/home/builder/aeternity/_build/test/lib/aehttp/test/aehttp_integration_SUITE.erl"},
          {line,5044}]},
     {aehttp_integration_SUITE,oracle_transactions,1,
         [{file,
              "/home/builder/aeternity/_build/test/lib/aehttp/test/aehttp_integration_SUITE.erl"},
          {line,2452}]},
     {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1546}]},
     {test_server,run_test_case_eval1,6,
         [{file,"test_server.erl"},{line,1062}]},
     {test_server,run_test_case_eval,9,
         [{file,"test_server.erl"},{line,994}]}]}
```

Now [tracked](https://www.pivotaltracker.com/story/show/163459427).

File: docs/release-notes/RELEASE-NOTES-1.4.0.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+[This][this-release] is a maintenance release.
+It:
+* Does all the things mentioned temporarily in files [/docs/release-notes/next/PT-*.md](/docs/release-notes/next/).
+
+TODO: When preparing the release, concatenate all `/docs/release-notes/next/*` Markdown files and place them in this file. (Hint: you can use auxiliary script `scripts/cat-files-in-directory-sorted-by-committer-date` and command `git log -p -w --color-moved`.)
+
+[this-release]: https://github.com/aeternity/aeternity/releases/tag/v1.4.0
+
+This release is backward compatible with `v1.3.*`, `v1.2.*`, `v1.1.*` and `v1.0.*`.
```
Comment (dincho) [inline docs/release-notes/RELEASE-NOTES-1.4.0.md]: In future can be changed to `This release is backward compatible with all `v1.*` releases.

**Pull request merged**